### PR TITLE
[Cosmos] Preview2

### PR DESCRIPTION
**Pull request by annatisch**

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: sdk/cosmos/azure-cosmos/samples/examples.py [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+# if no container with the specified ID was found in the database.
+# [START get_container]
+database = client.get_database_client(database_name)
+container = database.get_container_client(container_name)
+# [END get_container]
+
+# [START list_containers]
+database = client.get_database_client(database_name)
+for container in database.list_containers():
+    print(f"Container ID: {container.id}")
```
Comment (bryevdv) [inline sdk/cosmos/azure-cosmos/samples/examples.py]: f-string, is this expected to work with python < 3.6?

Comment (bryevdv) [reply sdk/cosmos/azure-cosmos/samples/examples.py]: Looks like three more lines below have as well

Comment (bryevdv) [reply sdk/cosmos/azure-cosmos/samples/examples.py]: also need future `print_function` if this is to work also on 2.7

Comment (annatisch) [reply sdk/cosmos/azure-cosmos/samples/examples.py]: Cool - I will use old-school formatting

**Pull request merged**